pavelib/assets.py

```diff
@@ -194,6 +194,17 @@ def get_theme_sass_dirs(system, theme_dir):
             ],
         })
 
+        # now compile theme sass files for certificate
+        if system == 'lms':
```

Comment:
Is it necessary to make it specific to the lms. I thought certificates are also rendered by Studio in preview mode. I could be wrong there.

Reply:
In my tests, only by setting the theme for the lms works also for preview. I guess the preview engine uses the lms rendering routines.